static/app/views/insights/pages/backend/laravelOverviewPage.tsx

```diff
@@ -434,22 +453,33 @@ function RequestsWidget({query}: {query?: string}) {
           ),
           timestamp: new Date(time).toISOString(),
         })),
-        field: `${codePrefix}xx`,
-        meta: firstSeries.meta!,
+        field,
+        meta: {
+          fields: {
+            [field]: 'integer',
+          },
+          units: {
+            [field]: null,
+          },
```

Comment:
👀 since you're using `events-stats` you don't need to construct your own meta, you can use the meta from the response! If the response for `count` isn't an integer, we need to fix that on the backend

Reply:
Roger! Although I wonder if you can do this using `equation|` syntax to calculate this on the backend 🤔